nvflare/private/fed/server/job_cmds.py

```diff
@@ -153,8 +163,9 @@ def delete_job(self, conn: Connection, args: List[str]):
 
     def abort_job(self, conn: Connection, args: List[str]):
         engine = conn.app_ctx
-        if not isinstance(engine, ServerEngine):
-            raise TypeError("engine must be ServerEngine but got {}".format(type(engine)))
+        job_runner = engine.job_runner
+        if not job_runner:
+            raise TypeError("engine missing job_runner")
```

Comment:
note that if this engine does not have `job_runner`
this L166 will raise AttributeError, we don't get to L167

Reply:
Will remove this check.